data/sitesFR.json

```diff
@@ -59,6 +59,25 @@
     "destination_main_page": "Accueil",
     "destination_search_path": "/index.php",
     "destination_content_path": "/"
+  },  
+  {
+    "id": "fr-gameofthrones",
+    "origins_label": "Game of Thrones Fandom Wiki",
+    "origins": [
+      {
+        "origin": "Game of Thrones Fandom Wiki",
+        "origin_base_url": "gameofthrones.fandom.com/fr",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "Wiki_Game_of_Thrones"
+      }
+    ],
+    "destination": "La Garde de Nuit Wiki",
+    "destination_base_url": "www.lagardedenuit.com",
```

Comment:
Since `www.lagardedenuit.com` is a parent site and `www.lagardedenuit.com/wiki` is the actual wiki, the `destination_base_url` should be `www.lagardedenuit.com/wiki`.

It looks like the `destination_content_path` for this wiki should be `/index.php?title=`.